net-fs/samba/files/samba-4.16.2-fix-musl-without-innetgr.patch

```diff
@@ -0,0 +1,23 @@
+# Gentoo bug 855047
```

Comment:
As discussed on IRC, please try upstream this

Reply:
I'll merge this but please take it upstream, and when you do, do a new PR to include the PR link in the patches (even if not merged yet)